shared/styles/colors.tsx

```diff
@@ -73,21 +73,117 @@ const colors = {
   red_75: 'rgba(255,0,0,0.75)',
   red_75_on_white: 'rgb(255,64,64)',
   transparent: 'rgba(0, 0, 0, 0)',
-  transparent_on_white: '#ffffff',
-  white: '#ffffff',
+  transparent_on_white: '#FFFFFF',
+  white: '#FFFFFF',
   white_0: 'rgba(255, 255, 255, 0)',
-  white_0_on_white: '#ffffff',
+  white_0_on_white: '#FFFFFF',
   white_20: 'rgba(255, 255, 255, 0.20)',
-  white_20_on_white: '#ffffff',
+  white_20_on_white: '#FFFFFF',
   white_40: 'rgba(255, 255, 255, 0.40)',
-  white_40_on_white: '#ffffff',
+  white_40_on_white: '#FFFFFF',
   white_75: 'rgba(255, 255, 255, 0.75)',
-  white_75_on_white: '#ffffff',
+  white_75_on_white: '#FFFFFF',
   white_90: 'rgba(255, 255, 255, 0.90)',
-  white_90_on_white: '#ffffff',
-  yellow: '#fff75a',
+  white_90_on_white: '#FFFFFF',
+  yellow: '#FFF75A',
   yellowDark: '#FFB800',
-  yellowLight: '#fffdcc',
+  yellowLight: '#FFFDCC',
 } as const
 
+export const darkColors: {[P in keyof typeof colors]: string | undefined} = {
+  black: 'rgba(255, 255, 255,  0.85)',
+  black_05: 'rgba(255, 255, 255,  0.05)',
+  black_05_on_white: 'rgb(13, 13, 13)',
+  black_10: 'rgba(255, 255, 255,  0.10)',
+  black_10_on_white: 'rgb(26, 26, 26)',
+  black_20: 'rgba(255, 255, 255,  0.20)',
+  black_20_on_white: 'rgb(51, 51, 51)',
+  black_35: 'rgba(255, 255, 255,  0.35)',
+  black_40: 'rgba(255, 255, 255,  0.40)',
+  black_50: 'rgba(255, 255, 255,  0.50)',
+  black_50_on_white: 'rgb(128, 128, 128)',
+  black_60: 'rgba(255, 255, 255,  0.60)',
+  black_63: 'rgba(255, 255, 255,  0.63)',
+  black_on_white: 'rgb(217, 217, 217)',
+  blue: '#B37100',
+  blueDark: '#C99C15',
+  blueDarker: '#EFC953',
+  blueDarker2: '#E7D291',
+  blueDarker2_75: 'rgba(231, 210, 145,  .75)',
+  blueDarker2_75_on_white: 'rgb(173, 157, 108)',
+  blueGrey: '#0D0B08',
+  blueGreyDark: '#1F1709',
+  blueLight: '#8C5900',
+  blueLighter: '#573300',
+  blueLighter2: '#140D03',
+  blueLighter3: '#080603',
+  blueLighter_20: 'rgba(87, 51, 0,  0.2)',
+  blueLighter_20_on_white: 'rgb(17, 10, 0)',
+  blueLighter_40: 'rgba(87, 51, 0,  0.4)',
+  blueLighter_40_on_white: 'rgb(35, 20, 0)',
+  blueLighter_60: 'rgba(87, 51, 0,  0.6)',
+  blueLighter_60_on_white: 'rgb(52, 31, 0)',
+  blue_30: 'rgba(204, 95, 0,  0.3)',
+  blue_30_on_white: 'rgb(63, 29, 0)',
+  blue_60: 'rgba(204, 95, 0,  0.6)',
+  blue_60_on_white: 'rgb(122, 57, 0)',
+  brown: 'rgb(184, 224, 238)',
+  brown_75: 'rgba(184, 224, 238,  0.75)',
+  brown_75_on_white: 'rgb(138, 168, 177)',
+  fastBlank: undefined,
+  green: '#C84266',
```

Comment:
Maybe not applicable yet, but I think green should stay green and red red, instead of swapping.

Reply:
yes, these are auto generated inverts, not the real colors